kafka/metrics_test.go

```diff
@@ -65,22 +63,16 @@ func TestProducerMetrics(t *testing.T) {
 		assert.NoError(t, rdr.Collect(context.Background(), &rm))
 
 		metrics := filterMetrics(t, rm.ScopeMetrics)
-		for i := range metrics {
-			t.Log(metrics[i].Name)
-		}
-		for i := range want {
-			if slices.Contains(ignore, want[i].Name) {
```

Comment:
I also refactored `ignored` metrics bit of the test here so it should fully address the concerns from https://github.com/elastic/apm-queue/issues/483